build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
 
     // Checker Framework Annotations
     implementation 'org.checkerframework:checker-qual:3.21.0'
+    implementation 'org.checkerframework:checker:3.21.0'
```

Comment:
This is a pretty old checker framework version - at least a year or two out of date. Is there a reason not to use the latest release (3.48.1)?